tests/src/FunctionalJavascript/CorporateSiteInfoSettingsFormTest.php

```diff
@@ -48,7 +48,12 @@ public function testConfigurationForm(): void {
     $page = $this->getSession()->getPage();
     $assert_session = $this->assertSession();
 
+    // Site owner and Content owners field is required.
+    $page->pressButton('Save configuration');
+    $assert_session->pageTextNotContains('The configuration options have been saved.');
```

Comment:
Doesn;'t the failed validation message specify something more clearly here? Otherwise we could be asserting a failed validation from a different field...

Reply:
It is FunctionalJavascript test, so it is not so easy to test html5 validation errors. I changed that to assert the validity of specific field values with using javascript. But it is maximum what we can do.
Switching to functional test (without javascript) we will lose asserting of some parts of functionality.

Reply:
For me, it is ok to rely on tests of the multivalue_form_element module (regarding adding more buttons).
But in this case, there is no more sense anymore to use the JavascriptFunctional test so I've converted the test to Functional.